regions/shapes/tests/test_common.py

```diff
@@ -8,7 +8,6 @@
 import pytest
 
 from ...core import PixCoord, BoundingBox
-from .utils import ASTROPY_LT_13, HAS_SHAPELY  # noqa
```

Comment:
Why was this removed?
CircleCI is complaining about the missing `HAS_SHAPELY.

Reply:
Oh! It was unintentional!! Thanks!